src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -204,7 +204,26 @@ def create_search_index(app, exception):
 
     search_index_list = []
 
-    for document in app.env.found_docs:
+    static_search_options = app.config.html_theme_options.get("static_search", {})
+    excluded_docs = static_search_options.get("files_to_exclude", [])
+
+    for exclude_doc in excluded_docs:
+        is_folder = exclude_doc.endswith("/")
```

Comment:
Wouldn't using `os.path.isdir(document)` be better ? I'm fearing that the ending with `"/"` might be OS dependent

Reply:
Yes, that would be the best, @Revathyvenugopal162. Thanks for pointing this out, @SMoraisAnsys. Does Sphinx treat these as actual paths, Revathy? If so, they may even have an `isdir()` method.

Reply:
> Yes, that would be the best, @Revathyvenugopal162. Thanks for pointing this out, @SMoraisAnsys. Does Sphinx treat these as actual paths, Revathy? If so, they may even have an `isdir()` method.

No, it is just the docnames, but we can try with output directory